client/src/components/client-area/OrderInProgress.vue

```diff
@@ -212,8 +260,9 @@
         <!-- invoice details -->
         <div
           v-if="
-            getStatus(customOrderBmsStatus, customOrderSellsyStatus)?.stepName === (StatusStep.Confirmed || StatusStep.InvoiceToBePaid) &&
-            orderDetails
+            orderDetails &&
+            (getStatus(customOrderBmsStatus, customOrderSellsyStatus)?.stepName === StatusStep.Confirmed ||
```

Comment:
You can use `[StatusStep.Confirmed, StatusStep.InvoiceToBePaid].includes(getStatus(customOrderBmsStatus, customOrderSellsyStatus)?.stepName)`. Makes it a bit easier to add or remove valide status.